pkgs/development/tools/tradcpp/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl }:
+{ stdenv, fetchurl, autoconf }:
```

Comment:
@edolstra I'm going to merge this as-is for now, but in general do you have a preference about autoreconfHook here? Since this one *only* needs autoconf it's not clear whether it's a big win.